rosidl_typesupport_opensplice_cpp/cmake/rosidl_typesupport_opensplice_cpp_generate_interfaces.cmake

```diff
@@ -119,12 +119,16 @@ foreach(_pkg_name ${rosidl_generate_interfaces_DEPENDENCY_PACKAGE_NAMES})
   endforeach()
 endforeach()
 
+set(_dds_idl_files_file "${_output_path}/dds_idl_files.txt")
+string(REPLACE ";" "\n" _dds_idl_files_concat "${_dds_idl_files}")
+file(WRITE "${_dds_idl_files_file}" ${_dds_idl_files_concat})
```

Comment:
Would `_dds_idl_files_lines` be a better name for the variable `_dds_idl_files_concat`?